mrp_production_request/README.rst

```diff
@@ -0,0 +1,99 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+MRP Production Request
+======================
+
+This module extends the functionality of Manufacturing to allow you to use
+Manufacturing Request (MR) as a previous step to Manufacturing Orders (MO).
+
+Configuration
+=============
+
+To configure this module to automatically generate Manufacturing Requests
+from procurement orders instead of directly create manufacturing orders yo
+need to:
+
+#. Go to the products that you want them to trigger manufacturing orders.
+#. Go to the *Inventory* tab.
+#. Check the route *manufacture* and the box *Manufacturing Request*.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Manufacturing > Manufacturing Requests*.
+#. Create a manufacturing request or open a existing one (assigned to you or
+   created from a procurement).
+#. If you click on *Request approval* button the user assigned as approver
+   will added to the thread.
+#. If you are the approver you can either click on *Approve* or *Reject*
+   buttons.
+#. Rejecting a MR will cancel associated procurements and propagate this
+   cancellation.
+#. Approving a MR will allow to create manufacturing orders.
+#. You can manually set to done a request by clicking in the button *Done*.
+
+To create MOs from MRs you have to:
+
+#. Go to approved manufacturing request.
+#. Click on the button *Create Manufacturing Order*.
+#. In the opened wizard, click on *Compute lines* so you will have a
+   quantity proposed for creating a MO. This quantity is the maximum quantity
+   you can produce with the current stock available in the source location.
+#. Use the proposed quantity or change it and click on *Create MO* at the
+   bottom of the wizard.
+
+**NOTE:** This module does not restrict the quantity that can be converted
```

Comment:
don't see a significative change in the layout of the README, am I missing something?? @elicoidal